swagger.yaml

```diff
@@ -4687,8 +4705,26 @@ definitions:
         type: boolean
         description: Schedule workloads on masters
         default: false
-      network_configuration:
-        $ref: '#/definitions/network_configuration'
+      cluster_networks:
+        type: array
+        description: Cluster networks that are associated with this cluster.
+        items:
+          type: object
+          $ref: '#/definitions/cluster_network'
```

Comment:
What should be the logic for update? If a record exists - is it add or update? If it does not exist - is it delete?

Reply:
Good question.
This is used in PATCH API - meaning the user doesn't have to pass the complete entity.

@avishayt thoughts on this?

Reply:
The user doesn't need to pass the entire cluster resource, but the simplest way to implement this is that they pass the entire list of networks and we replace any previous entry.  Let's not overcomplicate it.

Reply:
Updating the networks would work the same as it goes for the monitored
    operators.
    - In case the network weren't given (nil) then its update is ignored.
    - In case networks were provided - they are overriding the current
      cluster networks.